docs/faq.rst

```diff
@@ -93,6 +93,8 @@ tox 4 - CLI arguments changed
 - When you want to pass an option to a test command, e.g. to ``pytest``, now you must use ``--`` as a separator, this
   worked with version 3 also, but any unknown trailing arguments were automatically passed through, while now this is
   no longer the case.
+- Running ``--showconfig```or ``--help-ini`` with the ``-v`` flag will add debugging information. If you want to
```

Comment:
It's not tox ignore the verbose, just that happened to generate some extra lines at the start, but not afterwards.

Reply:
Right - it is getting late again - I'll update the pr tomorrow.